src/components/Shared/Filters/FilterListItemSelect.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  FormControl,
+  InputLabel,
+  ListItem,
+  MenuItem,
+  Select,
+} from '@material-ui/core';
+import React from 'react';
+import { Filter } from './Filter';
+
+interface Props {
+  filter: Filter;
+  value;
+  onUpdate: (value) => void;
+}
+
+export const FilterListItemSelect: React.FC<Props> = ({
+  filter,
+  value,
+  onUpdate,
+}: Props) => {
+  return (
+    <ListItem>
+      <FormControl style={{ flex: '1 1 auto' }}>
```

Comment:
```suggestion
      <FormControl style={{ flex: 'auto' }}>
```
https://developer.mozilla.org/en-US/docs/Web/CSS/flex#values
> This is equivalent to setting "flex: 1 1 auto".